.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

Comment:
I'd rather limit the noise on those topics, and I know dependabot can be quite pushy...

Are there "monthly" or "quarterly" options?

Reply:
```suggestion
      interval: "monthly"
```